app/views/job_applications/_form.html.erb

```diff
@@ -1,6 +1,6 @@
-<%= form_with(model: @job_posting, local: true) do |form| %>
-  <% if @job_posting.present? %>
-    <%= f.hidden_field :status, :value => 1 %>
-    <%= f.submit "Approve", class: "btn btn-large btn-success" %>
-  <% end %>
-<% end %>
\ No newline at end of file
+<%# <%= form_with(model: @job_posting, local: true) do |form| %>
+  <%# <% if @job_posting.present? %> %>
+    <%# <%= f.hidden_field :status, :value => 1 %> %>
```

Comment:
Is it worth deleting the whole file if it is all commented out?

Reply:
Removed